src/app/domains/wallets/domains.wallets.types.ts

```diff
@@ -9,4 +9,18 @@ export interface IWalletApiResponse {
 
 export interface IWalletPayload {
   name: string;
+}
+
+export interface IWalletDetailsApiResponse {
```

Comment:
I don't see such response for any of the endpoints: https://bycza-zagroda.github.io/track-expenses-app-documentation/#/Transactions/findAllTransactions

You're right it kinda would make sense to implement `GET /wallets/:id` to get such data in one request but because the number of wallets will be probably small I would just fetch all wallets like on wallets management page and find proper one by ID.
And fetch at the same time wallet's transactions by wallet ID. Wallet ID you can of course get from the URL.